README.md

```diff
@@ -1,9 +1,232 @@
 [<img alt="libfabric master branch Travis CI status" src="https://travis-ci.org/ofiwg/libfabric.svg?branch=master"/>](https://travis-ci.org/ofiwg/libfabric)
 [<img alt="libfabric Coverity scan suild status" src="https://scan.coverity.com/projects/4274/badge.svg"/>](https://scan.coverity.com/projects/4274)
 
-libfabric
-=========
+# libfabric
 
-The Open Fabrics Interfaces (OFI) is a framework focused on exporting fabric communication services to applications.
+The Open Fabrics Interfaces (OFI) is a framework focused on exporting fabric
+communication services to applications.
 
-See [the OFI web site](http://ofiwg.github.io/libfabric/) for more details, including a description and overview of the project, and detailed documentation of the libfabric APIs.
+See [the OFI web site](http://libfabric.org) for more details, including a
+description and overview of the project, and detailed documentation of the
+Libfabric APIs.
+
+## Installing libfabric
+
+On OS X the latest release of Libfabric can be installed using the
+[Homebrew](https://github.com/Homebrew/homebrew) package manager using the
+following command:
+
+```
+brew install libfabric
+```
+
```

Comment:
> Seems odd to have a section about "Installing libfabric" (which should be capital, BTW), but only talk about installing OS X. And then you have a section entitled "Building libfabric" (which should be capital) after that.

Right, I'll fix that. I was going to add more, but I wasn't sure if Libfabric is available in Linux distro repos.